.travis.yml

```diff
@@ -10,8 +10,10 @@ python:
 
 env:
     - DJANGO='1.8' REVERSION='1.9'  EXTRA=''
+    - DJANGO='1.8' REVERSION='1.10.1' EXTRA=''
     - DJANGO='1.9' REVERSION='1.10' EXTRA=''
     - DJANGO='1.8' REVERSION='1.9'  EXTRA='diff-match-patch'
+    - DJANGO='1.8' REVERSION='1.10.1'  EXTRA='diff-match-patch'
```

Comment:
I thought I read that Django 1.8 isn't compatible with 1.10, but I appear to have been mistaken.